pkg/cloudprovider/aws/ami.go

```diff
@@ -45,7 +45,7 @@ func NewAMIProvider(ssm ssmiface.SSMAPI, clientSet *kubernetes.Clientset) *AMIPr
 	}
 }
 
-func (p *AMIProvider) Get(ctx context.Context, constraints *Constraints, instanceTypes []cloudprovider.InstanceType) (string, error) {
+func (p *AMIProvider) getSSMParameter(ctx context.Context, constraints *Constraints, instanceTypes []cloudprovider.InstanceType) (string, error) {
```

Comment:
completely optional: I might move this function below `Get()`. I tend to follow a mixed pattern of DFS and "bigger concepts to the top"